codegen/gateway.go

```diff
@@ -125,7 +119,7 @@ func NewClientSpec(
 	if err := json.Unmarshal(instance.JSONFileRaw, &clientConfig); err != nil {
 		return nil, errors.Wrapf(
 			err,
-			"Could not parse class config json file %s: ",
+			"Could not parse class config json file: %s",
```

Comment:
Nit: better use `%q` instead of `: %s`, because the `:` is a separator for wrapped errors.